src/amo/containers/CategoryPage.js

```diff
@@ -9,6 +9,7 @@ import { apiAddonType } from 'core/utils';
 
 
 export function mapStateToProps(state, ownProps) {
+  const enableSearchSort = false;
```

Comment:
I guess overloading `mapStateToProps` for more than just state is OK but it seems a little weird. I think you can use composition to accomplish this a little cleaner:
````js
function CategoryPageBase(props) {
  return <SearchPage enableSearchSort={false} hasSearchParams {...props} />;
}

export default compose(
  ...,
  connect(mapStateToProps),
)(CategoryPageBase);
````